nuxeo-tools-hooks/nxtools/hooks/tests/webhooks/github_handlers/__init__.py

```diff
@@ -38,12 +38,19 @@ def __init__(self, filename):
 
         def __enter__(self):
             self.payload_file = open('nxtools/hooks/tests/resources/github_handlers/%s.json' % self.filename)
-            self.headers_file = open('nxtools/hooks/tests/resources/github_handlers/%s.headers.json' % self.filename)
-            return self.payload_file.read(), json.load(self.headers_file)
+            self.headers_file = None
```

Comment:
It seems that `self.headers_file` is only used in `__enter__()` and `__exit__()`?
What is the purpose of that attribute?

Reply:
The payload for the header part is useful for some tests (typically the ones relying on an event), and is used at least here for instance: https://github.com/nuxeo/nuxeo-tools-hooks/blob/release-1.1.0/nuxeo-tools-hooks/nxtools/hooks/tests/webhooks/github_handlers/test_jenkins_trigger.py#L44

Most of the time we do not need this file for tests, as we do not need headers: that's why i made this part optional, otherwise i was pushing useless .header.json files for each of the new json test content.